src/cf-typography/src/molecules/meta-header.less

```diff
@@ -1,8 +1,7 @@
 .m-meta-header {
-    padding-bottom: unit( 8px / @base-font-size-px, em );
+    padding-bottom: unit( 10px / @base-font-size-px, em );
     border-bottom: 1px solid @meta-header_border;
-    // TODO: Move margin to o-post-preview and other parent containers
-    margin-bottom: unit( 10px / @base-font-size-px, em );
+    overflow: auto;
```

Comment:
I'm not clear why this is necessary. Toggling it on and off doesn't seem to have any effect for me (testing in Firefox). But I'm not going to hold up the PR for a week to get an answer from Cat on it.

Reply:
I don't think this is serving any purpose either. Up to you to remove it or make a note and check back with Cat when she returns.